Assets/Scripts/Movement/MouseLook.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MouseLook : MonoBehaviour
```

Comment:
Is the mouse supposed to move the camera? Just a little confusing as to why there's no code in this script, unless you haven't implemented it yet.